ci/pkl-pipelines/pay-deploy/deploy-logging-pipeline.pkl

```diff
@@ -12,5 +12,9 @@ loggingEnvironments = new Listing<LoggingEnv> {
     account = "deploy" environment = "deploy-tooling"
     tf_root = "pay-infra/provisioning/terraform/deployments/deploy/deploy-tooling/management/logging_pipeline"
   }
+  new LoggingEnv {
+    account = "deploy" environment = "deploy-pay-cd"
```

Comment:
Minor, but I'd find it more typical if each attribute was on a new line, I see the previous env is the same so 🤷

Reply:
👍🏼 I will update it in the next PR